app/controllers/katello/api/v2/organizations_controller.rb

```diff
@@ -4,16 +4,22 @@ class Api::V2::OrganizationsController < ::Api::V2::OrganizationsController
 
     include Api::V2::Rendering
     include ForemanTasks::Triggers
-
-    before_action :local_find_taxonomy, :only => %w(repo_discover cancel_repo_discover
-                                                    download_debug_certificate cdn_configuration
-                                                    redhat_provider update releases)
+    LOCAL_FIND_TAXONOMY_ACTIONS = %w(repo_discover cancel_repo_discover download_debug_certificate
+                                     cdn_configuration redhat_provider update releases).freeze
+    before_action :local_find_taxonomy, :only => LOCAL_FIND_TAXONOMY_ACTIONS
 
     resource_description do
       api_version 'v2'
       api_base_url "/katello/api"
     end
 
+    def self.delay_find_taxonomy_actions
```

Comment:
```suggestion
    def self.before_find_taxonomy_actions
```
will read better when used:

```
        Katello::Api::V2::OrganizationsController.before_find_taxonomy_actions do
          Katello::Api::V2::OrganizationsController.add_smart_proxy_filters(
            [:index, :download_debug_certificate],
            features: ForemanRhCloud.on_prem_smart_proxy_features
          )
        end
```